src/compton/test/t2Compton.cc

```diff
@@ -0,0 +1,322 @@
+//-----------------------------------*-C++-*----------------------------------//
+/*!
+ * \file   compton/test/t2Compton.cc
+ * \author Andrew Till
+ * \date   2020 Oct 14
+ * \brief  Implementation file for t2Compton
+ * \note   Copyright (C) 2017-2020 Triad National Security, LLC.
+ *         All rights reserved. */
+//------------------------------------------------------------------------------------------------//
+
+#include "c4/ParallelUnitTest.hh"
+#include "compton/Compton.hh"
+#include "ds++/Release.hh"
+#include "ds++/Soft_Equivalence.hh"
+#include <fstream>
+#include <iomanip>
+#include <sstream>
+#include <stdexcept>
+#include <vector>
+
+namespace rtt_compton_test {
+
+using rtt_dsxx::soft_equiv;
+
+//------------------------------------------------------------------------------------------------//
+// TESTS
+//------------------------------------------------------------------------------------------------//
+
+//!  Tests the Compton constructor and a couple of access routines.
+void compton_file_test(rtt_dsxx::UnitTest &ut) {
+
+  // Make true if golds need updating
+  const bool do_print = false;
+
+  // Tolerance used for checks
+  const double tol = 1e-11;
+
+  // Start the test.
+
+  std::cout << "\n---------------------------------------------------------\n"
+            << "   Test Draco code calling CSK_generator routines\n"
+            << "---------------------------------------------------------\n";
+
+  // open a small mg opacity file:
+  const std::string filename = ut.getTestSourcePath() + "../../compton2/test/dummy_data";
```

Comment:
I wanted to be sure that compton and compton2 were getting the same answer for the same input, which is why I wrote this test. They are. Since compton is already tested, should I just remove this test? It's using the data in compton2 but the methods in compton.

Reply:
You can leave it here.  I just wanted to make sure that this design was intentional.